### ISSUE TITLE
HMR not working

### ISSUE THREAD

**Issue by afm-sayem** [sveltejs/sapper-template#67]
Changing any file generates following error:
```
Uncaught (in promise) Error: Manifest request to client/4b956183ae495052a330.hot-update.json timed out.
    at XMLHttpRequest.request.onreadystatechange
```
I added a timeout according to [this comment](https://github.com/sveltejs/sapper/issues/277#issue-328777707), but I'm still getting the error. Changes are not appearing.

**Comment by Rich-Harris** [comment]
Can you try again with the latest version? It should be fixed now

**Comment by benfavre** [comment]
FYI HMR is still not fixed.

**Comment by samuelhnrq** [comment]
Was this fixed by sveltejs/svelte#3148? Should we update the comment in the webpack config?

**Comment by antony** [comment]
Closing in favour of https://github.com/sveltejs/sapper/issues/764
